packages/react/webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const fs = require("fs");
+const path = require("path");
+const webpack = require("webpack");
+const WriteFilePlugin = require("write-file-webpack-plugin");
+
+const JS_DR = path.resolve(__dirname, "examples/js");
+
+
+function buildEntries() {
+	return fs.readdirSync(JS_DR).reduce((entries, dir) => {
+		if (dir === "dat.gui.js") {
+			return entries;
+		}
+		entries[dir] = path.join(JS_DR, dir);
+
+		return entries;
+	}, {});
+}
+
+
+module.exports = {
+	devtool: "inline-source-map",
+	entry: buildEntries(),
+	output: {
+		filename: "[name]",
```

Comment:
@younkue It's some tiny things. I would be better to match indentation with others.